std/algorithm/sorting.d

```diff
@@ -2308,6 +2308,13 @@ unittest
     topN(a, b);
     sort(a);
     assert(a == [0, 1, 2, 2, 3]);
+
+    // bug 15420
```

Comment:
Well, that explains the /// before the block.  Sorry, I'm not too familiar with ddoc.